packages/samples/sudt/src/response.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @module src/response
+ * @description
+ * This module defines the response format of the application.
+ * It is used to wrap the response data and return to the client.
+ */
+
+export class SudtResponse<T> {
+  code: string;
+  data: T;
+
+  constructor(code: string, data: T) {
+    this.code = code;
+    this.data = data;
+  }
+
+  static ok<T>(data: T) {
+    return new SudtResponse('200', data);
+  }
+
+  static err<T>(data: T, code: string) {
+    return new SudtResponse(code, data);
+  }
+}
```

Comment:
I think maybe we can add it as a default response for developers to choose to the framework.

Reply:
How would it be used if it's a default response

Reply:
> How would it be used if it's a default response

I think we can define a default response type like this, and then developers could use it directly without designing another response class.

Reply:
> > How would it be used if it's a default response
> 
> I think we can define a default response type like this, and then developers could use it directly without designing another response class.

You may refer to a `BaseError` to extend, and the framework will handle `error` extended from `BaseError` automatically, e.g. output formatted error message, submit error to monitor, etc.

Reply:
> I copied the `Response` from mvp-dapp. Shall we support the default response struct? If so, we'd better add a default middleware to transfer the response.

It's a good idea but requires some work, maybe we should open an issue and add this feature in another PR

Reply:
> maybe we should open an issue and add this feature in another PR

I agree with that.

Reply:
Tracked by issue https://github.com/ckb-js/kuai/issues/491